alicloud/resource_alicloud_vpn_route_entry.go

```diff
@@ -0,0 +1,219 @@
+package alicloud
+
+import (
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/vpc"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+	"time"
+)
+
+func resourceAliyunVpnRouteEntry() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAliyunVpnRouteEntryCreate,
+		Read:   resourceAliyunVpnRouteEntryRead,
+		Update: resourceAliyunVpnRouteEntryUpdate,
+		Delete: resourceAliyunVpnRouteEntryDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"vpn_gateway_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"next_hop": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+
+			"route_dest": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+
+			"weight": {
+				Type:         schema.TypeInt,
+				Required:     true,
+				ValidateFunc: validateVpnBandwidth([]int{0, 100}),
+			},
+
+			"publish_vpc": {
+				Type:     schema.TypeBool,
+				Required: true,
+			},
+
+			// Computed values
+			"create_time": {
+				Type:     schema.TypeFloat,
+				Computed: true,
+			},
+
+			"old_weight": {
+				Type:     schema.TypeInt,
+				Computed: true,
```

Comment:
I think there is no need to output the extra parameter `create_time`, `old_weigth` and `state`